k8s-deploy/examples/party-10000/cluster.yaml

```diff
@@ -29,6 +29,10 @@ storage: Eggroll
 algorithm: Basic
 device: CPU
 
+# you can customize some keys which will be ignored in yaml validation
+skippedKeys:
+  - route_table
+
```

Comment:
What will happen if we don't add these lines?

Reply:
Since "route_table" has already been hardcoded into golang code, no difference would be made if we don't add these line in config. But I want to tell users there could be some skippedkeys.